default-config.toml

```diff
@@ -87,12 +87,60 @@ emoji_favicons = false
 
 
 [keybindings]
-# In the future there will be more settings here.
-
+# If you have a non-US keyboard, use bind_tab1 through bind_tab0 to
+# setup the shift-number bindings: Eg, for US keyboards (the default):
+#bind_tab1 = "!"
+#bind_tab2 = "@"
+#bind_tab3 = "#"
+#bind_tab4 = "$"
+#bind_tab5 = "%"
+#bind_tab6 = "^"
+#bind_tab7 = "&"
+#bind_tab8 = "*"
+#bind_tab9 = "("
+#bind_tab0 = ")"
+# Also, shift_numbers still works:
 # Hold down shift and press the numbers on your keyboard (1,2,3,4,5,6,7,8,9,0) to set this up.
-# It is default set to be accurate for US keyboards.
-shift_numbers = "!@#$%^&*()"
-
+# EG, for US keyboards (the default):
+#shift_numbers = "!@#$%^&*()"
+
+# Notes:
+# * whitespace is not allowed in any of the keybindings! Use 'Space' and 'Tab' to bind to those keys.
+# * Multiple keys can be bound to one command, just use a TOML array.
+# * To add the Alt modifier, the binding must start with Alt-, should be reasonably universal
+# * Ctrl- isn't a universal modifier, just the keys listed in tcell.KeyNames, see:
+# https://github.com/gdamore/tcell/blob/master/key.go (it was line 83 when this comment was written)
+
+# And the rest of the default keybindings
```

Comment:
I don't think it's necessary to list all the default keybindings here. See how the `[theme]` section is done, where all the keys are listed, and some are explained, but not all the values. So maybe explain `bind_edit`, the `bind_link*` ones, but for the rest just list them.